node.js_mongodb/SingleUpload.js

```diff
@@ -0,0 +1,24 @@
+const DBHelper=require('./DB_helper');
+const readline = require('readline');
+PromptAndUpload();
+
+async function PromptAndUpload() {
+    const rl = readline.createInterface({
```

Comment:
Let's use a more descriptive name than r1. I have no clue what this is supposed to be.